src/transeq.f90

```diff
@@ -450,6 +452,27 @@ subroutine momentum_rhs_eq(dux1,duy1,duz1,rho1,ux1,uy1,uz1,ep1,phi1,divu3)
        if (nrank == 0) print *,'TRIPPING!!'
     endif
 
+    ! Monitor velocity and pressure gradients at the first sub-iteration only
```

Comment:
This is mixing functionality - writing to probes should be performed elsewhere such as the "postproc" subroutines